src/rgw/rgw_rest_swift.cc

```diff
@@ -1666,7 +1666,10 @@ void RGWInfo_ObjStore_SWIFT::list_swift_data(Formatter& formatter,
 
   string ceph_version(CEPH_GIT_NICE_VER);
   formatter.dump_string("version", ceph_version);
-  formatter.dump_int("max_meta_name_length", 81);
```

Comment:
This were causing failures in the container-related tests of Swift (e.g. `test.functional.test_container:TestContainer.test_POST_bad_metadata`). The patch rectifies that as `/info` returns `80` for `max_meta_name_length` now.